src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -5063,4 +5061,112 @@ export class ResultsViewPageStore {
             return isMixedReferenceGenome(this.studies.result);
         }
     }
+
+    readonly clinicalTrackOptions = remoteData({
+        await: () => [
+            this.clinicalAttributes,
+            this.clinicalAttributeIdToAvailableFrequency,
+            this.clinicalAttributes_customCharts,
+        ],
+        invoke: () => {
+            const uniqueAttributes = _.uniqBy(
```

Comment:
we should really move this logic to helper files.  putting all this inside the invoke is one reason by this file is >5000 lines long.   also it can't be tested